openquakeplatform/vulnerability/templates/admin/vulnerability/object_history.html

```diff
@@ -0,0 +1,44 @@
+{% extends "admin/vulnerability/base_site.html" %}
+{% load i18n %}
+{% load admin_urls %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+mop - object_history
```

Comment:
text for debug, please remove it

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/b1ceaa2bd408b1e6303a1ced1975e52a7fcfce72